pkg/container-collection/options.go

```diff
@@ -585,6 +581,30 @@ func WithLinuxNamespaceEnrichment() ContainerCollectionOption {
 	}
 }
 
+// WithOCIConfigEnrichment enriches container using provided OCI config
+func WithOCIConfigEnrichment() ContainerCollectionOption {
+	return func(cc *ContainerCollection) error {
+		cc.containerEnrichers = append(cc.containerEnrichers, func(container *Container) bool {
+			if container.OciConfig != nil {
```

Comment:
Shouldn't we verify that container is not already enriched as we are doing [here](https://github.com/inspektor-gadget/inspektor-gadget/blob/f6b8c02bd5d1515e022fbf8cec220e7fad09d57f/pkg/container-collection/options.go#L67-L71)? BTW, we should also verify if the Pod UID and the Runtime.